core/src/main/java/org/apache/carbondata/core/util/CarbonTimeStatisticsFactory.java

```diff
@@ -17,47 +17,51 @@
 
 package org.apache.carbondata.core.util;
 
-import org.apache.carbondata.core.carbon.querystatistics.DriverQueryStatisticsRecorder;
+import org.apache.carbondata.core.carbon.querystatistics.*;
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
 
 public class CarbonTimeStatisticsFactory {
   private static String LoadStatisticsInstanceType;
-  private static LoadStatistics LoadStatisticsInstance;
-  private static DriverQueryStatisticsRecorder QueryStatisticsRecorderInstance;
+  private static String queryStatisticsRecorderInstanceType;
 
   static {
     CarbonTimeStatisticsFactory.updateTimeStatisticsUtilStatus();
-    LoadStatisticsInstance = genLoadStatisticsInstance();
-    QueryStatisticsRecorderInstance = genQueryStatisticsRecorderInstance();
   }
 
   private static void updateTimeStatisticsUtilStatus() {
     LoadStatisticsInstanceType = CarbonProperties.getInstance()
         .getProperty(CarbonCommonConstants.ENABLE_DATA_LOADING_STATISTICS,
             CarbonCommonConstants.ENABLE_DATA_LOADING_STATISTICS_DEFAULT);
-  }
-
-  private static LoadStatistics genLoadStatisticsInstance() {
```

Comment:
dont delete this function，it is used for gen instance only once，but you can use if else to replace swith here.